sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/NetworkStateHelper.java

```diff
@@ -233,19 +226,19 @@ private synchronized void handleNetworkStateUpdate() {
      * @param connected whether the network is connected or not.
      */
     private void notifyNetworkStateUpdated(boolean connected) {
+        AppCenterLog.debug(LOG_TAG, "Network has been " + (connected ? "connected." : "disconnected."));
```

Comment:
@guperrot It's logged here. Anyway, I'll revert the logs in the callbacks

Reply:
I couldn't see any debug logs when I was testing, will retry.

Reply:
I see this log now, maybe I missed it last time.